mirage/config.js

```diff
@@ -1,13 +1,23 @@
 import ENV from '../config/environment';
 
+let realEnpoints = ['jobs'];
```

Comment:
A list of endpoints to pass through to the real API (unless the environment is 'test')

Reply:
In the test environment, are we still able access gatekeeper routes? If so where does gatekeeper create its token if its not coming from the back end?